### PR TITLE
Update install.rst: Add cv2 error to faq

### PR DESCRIPTION
**Pull request by aafkevandenberg**
I included a method to resolve the error that cv2 cannot be found. This solution is the first suggestion on the opencv website, https://pypi.org/project/opencv-python/, and worked for at least one customer.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -304,6 +304,18 @@ If you have already activated the correct environment, but you still do not see
...
 
+**When importing lumicks.pylake on Windows I get an error with cv2**
+
+The full error message is::
+
+    ImportError: DLL load failed while importing cv2: The specified procedure could not be found.
+    
+To resolve this error, follow the following steps:
+    
+* Remove the current Pylake installation. If you are not using Anaconda for anything else but Pylake, you can simply remove Anaconda in this step.
```
Comment (JoepVanlier) [inline docs/install.rst:315]: Are we sure removing only Pylake is sufficient?

Comment (aafkevandenberg) [reply docs/install.rst:315]: No.... I will just say to remove Anaconda then.

**Review by JoepVanlier — APPROVED** [review]
Looks good to go. Hope this resolves most of these issues in the field. 🐍

File: docs/install.rst (hunk 1 above)
Comment (JoepVanlier) [reply docs/install.rst]: 👍

**Pull request merged**